prow/scripts/cluster-integration/kyma-integration-gardener.sh

```diff
@@ -103,20 +103,19 @@ gardener::generate_overrides
 export CLEANUP_CLUSTER="true"
 gardener::provision_cluster
 
-if [[ "${KYMA_MAJOR_VERSION}" == "2" ]]; then
+kyma::deploy_kyma \
+  -p "$EXECUTION_PROFILE" \
+  -d "$KYMA_SOURCES_DIR"
+if [[ "${KYMA_DELETE}" == "true" ]]; then
+  sleep 30
+  kyma::undeploy_kyma
+  sleep 30
   kyma::deploy_kyma \
-    -p "$EXECUTION_PROFILE" \
-    -d "$KYMA_SOURCES_DIR"
-  if [[ "${KYMA_DELETE}" == "true" ]]; then
-    sleep 30
-    kyma::undeploy_kyma
-    sleep 30
-    kyma::deploy_kyma \
-       -p "$EXECUTION_PROFILE" \
-       -d "$KYMA_SOURCES_DIR"
-  fi
+      -p "$EXECUTION_PROFILE" \
+      -d "$KYMA_SOURCES_DIR"
+fi
 # this will be extended with the next components
```

Comment:
@kyma-project/goat can you please review the changes here.

Reply:
LGTM